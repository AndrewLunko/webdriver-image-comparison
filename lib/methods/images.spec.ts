// import {join} from 'path';
// import {pathExistsSync, removeSync} from 'fs-extra';
// import {checkBaselineImageExists} from './images';
//
// describe('images', () => {
//   const baselineFolder = join(process.cwd(), '/.tmp/baselineFolder');
//
//   afterEach(() => removeSync(baselineFolder));
//
//   describe('checkBaselineImageExists', () => {
//     it('should check if the baseline image exists and not create a new one', async () => {
//       let actualFilePath = join(process.cwd(), '/package.json');
//       const baselineFilePath = `${baselineFolder}/package.json`;
//       const autoSaveBaseline = false;
//
//       expect(pathExistsSync(baselineFilePath)).toMatchSnapshot();
//       await expect(checkBaselineImageExists(actualFilePath, baselineFilePath, autoSaveBaseline)).rejects.toContain(
//         `Baseline image not found, save the actual image manually to the baseline.`);
//       expect(pathExistsSync(baselineFilePath)).toMatchSnapshot();
//     });
//
//     it('should check if the baseline image exists and create a new one', async () => {
//       let actualFilePath = join(process.cwd(), '/package.json');
//       const baselineFilePath = `${baselineFolder}/package.json`;
//       const autoSaveBaseline = true;
//
//       expect(pathExistsSync(baselineFilePath)).toMatchSnapshot();
//       await checkBaselineImageExists(actualFilePath, baselineFilePath, autoSaveBaseline);
//       expect(pathExistsSync(baselineFilePath)).toMatchSnapshot();
//     });
//   });
// });
