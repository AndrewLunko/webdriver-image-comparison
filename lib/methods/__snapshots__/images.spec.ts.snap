// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`images checkBaselineImageExists should check if the baseline image exists and create a new one 1`] = `false`;

exports[`images checkBaselineImageExists should check if the baseline image exists and create a new one 2`] = `true`;

exports[`images checkBaselineImageExists should check if the baseline image exists and not create a new one 1`] = `false`;

exports[`images checkBaselineImageExists should check if the baseline image exists and not create a new one 2`] = `false`;
